src/universal/redux/makeReducer.js

```diff
@@ -11,13 +11,27 @@ import outcomeCardReducer from 'universal/modules/outcomeCard/ducks/outcomeCardD
 
 const {SET_SUBMIT_SUCCEEDED} = actionTypes;
 
-const formPlugin = {
-  agendaInput: (state, action) => {
-    // wipe the value clean when submitted
-    return (action.type === SET_SUBMIT_SUCCEEDED) ? undefined : state;
+// wipe the value clean when submitted
+const wipeAfterSuccess = (formToClear) => (state, action) => {
+  if (action.type === SET_SUBMIT_SUCCEEDED) {
+    if (action.meta.form === formToClear) {
+      return undefined;
+    }
   }
+  return state;
 };
 
+const formPluginFactory = () => {
+  // add new fields to this array if you want em cleared
+  const clearMeAfterSubmit = ['agendaInput', 'inviteTeamMember'];
```

Comment:
It's very magical. I'd prefer the redux action because it's more explicit. Failing that, a comment should be added to the submit handlers of each of these forms so future selves can remember where we put the magic.